test/Microsoft.OpenAPI.OData.Reader.Tests/Resources/TripService.OpenApi.V2.json

```diff
@@ -15361,12 +15258,7 @@
         ],
         "summary": "Invoke action GetPeersForTrip",
         "operationId": "NewComePeople.Person.GetPeersForTrip",
-        "consumes": [
```

Comment:
Are we expecting to delete these?

Reply:
The serialization to v2 has its shortcomings when it comes to reference request bodies. I am working on a fix on the OpenAPI.NET library.